### PR TITLE
Fix and more dolibarize Stripe

### PR DESCRIPTION
**Pull request by ptibogxiv**
Dolibarize and move file
update php stripe library
add list of dispute

### CODE REVIEW

File: htdocs/compta/paiement/card.php [hunk 1]
```diff
@@ -308,6 +312,22 @@
...
 
+// External link
+if (!empty($object->ext_payment_id) && !empty($object->ext_payment_site) ) { 
+
+$stripe=new Stripe($db);
+$stripeacc = $stripe->getStripeAccount($object->ext_payment_site);
+
+if (!empty($stripeacc)) $connect=$stripeacc.'/';	
+  	$url='https://dashboard.stripe.com/'.$connect.'test/payments/'.$object->ext_payment_id;
+			if ($object->payment_site == StripeLive)
```
Comment (eldy) [inline htdocs/compta/paiement/card.php]: You mean ->ext_payment_site here i think ?

Comment (ptibogxiv) [reply htdocs/compta/paiement/card.php]: @eldy  yes, i fix it immediately

File: htdocs/compta/paiement/card.php [hunk 2]
```diff
@@ -197,6 +199,7 @@
 if ($action == 'delete')
 {
 	print $form->formconfirm($_SERVER['PHP_SELF'].'?id='.$object->id, $langs->trans("DeletePayment"), $langs->trans("ConfirmDeletePayment"), 'confirm_delete','',0,2);
+
```
Comment (eldy) [inline htdocs/compta/paiement/card.php]: Adding spaces here generate code syntax errors in continuous integration test

**Comment by ptibogxiv** [comment]
close because new PR with new payment intent method

**Pull request closed without merging**